languages/cpp/exercises/concept/numbers/.docs/hints.md

```diff
@@ -0,0 +1,12 @@
+## General
+
+- Determining the success rate can be done through a [conditional statement][if-statement].
+- C++ allows for multiplication to be applied to two different number types (such as an `int` and a `double`). It will automatically return the "largest" data type.
+- Ranges of integer and floating point types listed in [fundamental types][fundamental-types]
+- Fixed-width integer types, such as `std::int32_t`, are listed in [fixed width integer types][fixed-width-integer-types].
+- Information about types and how to acquire it during compilation is available in [numeric limits library][numeric-limits].
```

Comment:
The hints are nice! Are there hints that can be linked to a specific task? See [this file](https://github.com/exercism/v3/blob/master/languages/cpp/exercises/concept/strings/.docs/hints.md) for an example on how to link specific hints to specific tasks.